src/app/client/src/app/modules/learn/components/course-consumption/course-consumption-header/course-consumption-header.component.html

```diff
@@ -70,9 +70,10 @@ <h6 class="ui small info message" *ngIf="enrolledCourse && getBatchStatus() && b
 
       <div class="sb-g-col-xs-12 sb-g-col-md-3 sb-g-col-lg-3 sb-g-col-xxxl-4 certified-course">
 
-        <div class="certified-course__btn" *ngIf="showAddGroup">
+        <div class="certified-course__btn">
           <button class="sb-btn sb-btn-secondary sb-btn-normal ml-auto textbook__addbtn"
-            (click)="addActivityToGroup(); this.logTelemetry('add-to-group', courseHierarchy)">{{resourceService?.frmelmnts?.lbl?.AddtoGroup}}</button>
+          appAddToGroup [identifier]="courseHierarchy?.identifier" [pageId]="courseHierarchy?.contentType.toLowerCase()">
+          {{resourceService?.frmelmnts?.lbl?.AddtoGroup}}</button>
```

Comment:
Inform Ahment/Anil to change this "pageId" to "contextType"

Reply:
sure